zerver/openapi/zulip.yaml

```diff
@@ -3018,6 +3018,428 @@ paths:
+        required: false
+      - name: add_emoji_by_admins_only
+        in: query
+        description: |
+          A boolean that indicates if only admins can add emojis or not.
+          If false, it means that admins and members can add emojis.
+        schema:
+          type: boolean
+          nullable: true
+        example: false
+        required: false
+      - name: allow_message_deleting
+        in: query
+        description: |
+          A boolean that indicates if message deleting is allowed.
+        schema:
+          type: boolean
+          nullable: true
+        example: true
+        required: false
+      - name: message_content_delete_limit_seconds
+        in: query
+        description: |
+          Time, in seconds, after posting for which deleting a message is allowed.
+          Admins can delete messages at any time.
+        schema:
+          type: integer
+          nullable: true
+        example: 120
+        required: false
+      - name: allow_message_editing
+        in: query
+        description: |
+          A boolean indicating whether message editing is allowed.
+        schema:
+          type: boolean
+          nullable: true
+        example: true
+        required: false
+      - name: allow_community_topic_editing
+        in: query
+        description: |
+          A boolean indicating if users can edit the topic of any message.
+        schema:
+          type: boolean
+          nullable: true
+        example: true
+        required: false
+      - name:  mandatory_topics
+        in: query
+        description: |
+          A boolean indicating whether is mandatory that all stream messages
+          have topics.
+        schema:
+          type: boolean
+          nullable: true
+        example: true
+        required: false
+      - name: message_content_edit_limit_seconds
+        in: query
+        description: |
+          Time, in seconds, after posting for which editing a message is allowed.
+          Admins can delete messages at any time.
+        schema:
+          type: integer
+          nullable: true
+        example: 120
+        required: false
+      - name: allow_edit_history
+        in: query
+        description: |
+          A boolean indicating whether it is allowed to edit history.
+        schema:
+          type: boolean
+          nullable: true
+        example: true
+        required: false
+      - name: default_language
+        in: query
+        description: The default language of the realm.
+        schema:
+          type: string
+          nullable: true
+        example: en-gb
+        required: false
+      - name: waiting_period_threshold
+        in: query
+        description: |
+          Waiting period, in days, before new members turn into full members.
+        schema:
+          type: integer
+          nullable: true
+        example: 3
+        required: false
+      - name: authentication_methods
+        type: object
```

Comment:
The type should be inside `schema`. Also all `array` and `object` parameters should have an example.